src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -198,6 +198,7 @@ def load_arguments(self, _):
         c.argument('skip_subnet_role_assignment', action='store_true')
         c.argument('api_server_authorized_ip_ranges', type=str, validator=validate_ip_ranges)
         c.argument('attach_acr', acr_arg_type)
+        c.argument('nodepool_tags', nargs='*', validator=validate_nodepool_tags, help='space-separated tags: key[=value] [key[=value] ...]. Use "" to clear existing tags.')
```

Comment:
I think you can use `tags_type` directly and overwrite the `help=` message.

Reply:
No, I tried to use tags_type, it won't work. Since the validator for tags_type, will read ns.tags, not ns.nodepool_tags, so I re-write a new function in our owned subfolder.

Reply:
@MyronFanQiu please help to merge if no other concerns or questions :)

Reply:
@MyronFanQiu When I run test, it failed with error, can you shed light on how to fix it?

```
AssertionError: No match for the request (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000001?api-version=2020-01-01>) was found. Can't overwrite existing cassette ('/Users/qizhe/azure-cli/azure-cli/src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_nodepool_create_scale_delete.yaml') in your current record mode ('once').
```

Reply:
@zqingqing1 Hi. You should run it with `--live` like `azdev test acs --live`. Could you please a single test for this new feature?

Reply:
Surely, will add it.

Reply:
when I test it locally, it passed, but https://dev.azure.com/azure-sdk/public/_build/results?buildId=261822&view=logs&j=ef7e37da-9721-5d17-b890-4759c548b596&t=ce7f2a4a-7f31-5156-db1b-13049c09b684&l=7799 test failed with "Can't overwrite existing cassette". How gonna we fix it?

Reply:
The API version is not consistent. Do you need to bump api-version? Looks like you install the package locally.

Reply:
You are right, aks-preview are using new api-version. I am testing with this cli extension, after removal, test should pass.

Reply:
test passed, please help to merge this PR. Appreciated.